monitoring/alert/alert_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	monitoring "cloud.google.com/go/monitoring/apiv3"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"golang.org/x/net/context"
+	"google.golang.org/api/iterator"
+	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+)
+
+func TestListAlertPolicies(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
+	if err := listAlertPolicies(buf, tc.ProjectID); err != nil {
+		t.Fatalf("listAlertPolicies got error: %v", err)
+	}
+	want := "Display Name"
+	if got := buf.String(); !strings.Contains(got, want) {
```

Comment:
doesn't matter, but i kinda like this:

```
if got, want := buf.String(), "Display Name"
``

maybe you do too. if not, no problem

Reply:
It's faster for me to parse & understand as separate declarations. So, leaving as is.